services/core-web/common/actionCreators/mineActionCreator.js

```diff
@@ -355,3 +355,87 @@ export const deleteMineComment = (mineGuid, commentGuid) => (dispatch) => {
       throw new Error(err);
     });
 };
+
+export const fetchMineAlertByMine = (mineGuid) => (dispatch) => {
```

Comment:
We may want to update this to fetchMineAlertsByMine to avoid confusion and be consistent with the other constants in use. Not a huge thing, dealer's choice.